modules/add.js

```diff
@@ -0,0 +1,24 @@
+import renderBookList from './booklist.js';
+
+const bookCollection = []; // Declare and initialize the bookCollection array
+
+// Function to add a new book to the collection
+function addBook(title, author) {
```

Comment:
- Hey, please, kindly use the ES6 arrow function since you're working on learning to use the ES6 syntax.